src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/config/ServiceBusConfiguration.java

```diff
@@ -1,98 +1,47 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.config;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.microsoft.azure.servicebus.IMessageHandler;
 import com.microsoft.azure.servicebus.IQueueClient;
-import com.microsoft.azure.servicebus.MessageHandlerOptions;
-import com.microsoft.azure.servicebus.QueueClient;
-import com.microsoft.azure.servicebus.ReceiveMode;
-import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
-import com.microsoft.azure.servicebus.primitives.ServiceBusException;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Lazy;
-import uk.gov.hmcts.reform.bulkscanprocessor.services.ErrorNotificationService;
+import org.springframework.context.annotation.Profile;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.MessageAutoCompletor;
 import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.ServiceBusHelper;
-import uk.gov.hmcts.reform.bulkscanprocessor.tasks.ErrorNotificationHandler;
-
-import java.time.Duration;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import javax.annotation.PostConstruct;
 
 @Lazy
 @Configuration
+@AutoConfigureAfter(QueueClientConfig.class)
+@Profile("!nosb") // no servicebus queue handler registration
 public class ServiceBusConfiguration {
 
-    private static final ExecutorService QUEUE_READ_EXEC = Executors.newSingleThreadExecutor(r ->
-        new Thread(r, "notifications-queue-read")
-    );
-
     @Autowired
     private ObjectMapper objectMapper;
 
-    @Autowired
-    private ErrorNotificationService errorNotificationService;
-
-    @Value("${queues.read-notifications.enabled}")
-    private boolean readNotificationsEnabled;
 
-    @Value("${queues.read-notifications.connection-string}")
-    private String readNotificationConnectionString;
-
-    @Value("${queues.read-notifications.queue-name}")
-    private String readNotificationQueueName;
-
-    @Bean(name = "envelopes")
```

Comment:
missing `envelopes-helper` bean name - it is used in `OrchestratorNotificationTask`

Reply:
It is right in the line below.

Helper aside, maybe I should change the bean name for queue client to `envelopes-client` or sth?

Reply:
ah, must've crossed eyes and missed it

since you sifting around servicebus configs, worth naming/renaming those clients too ;)